build/syntax-highlight.js

```diff
@@ -58,7 +58,7 @@ function syntaxHighlight($, doc) {
   $("pre[class*=brush]").each((_, element) => {
```

Comment:
This is an exciting and welcome change, but I'm concerned that it will change the meaning of some content.

If I understand this correctly, this will attach the copy-to-clipboard widget to every code block, including those meant to illustrate something negative (i.e., those with the class `example-bad`, [for example](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Unexpected_token#expression_expected)).

It might be nice to exclude those code blocks, otherwise we'll be sending a mixed message: don't do this, even though we've made easy for you to do this.

Reply:
> We could skip injecting clipboard buttons to certain `<pre>` tags with a specific class like `.example-bad`.

Yes, this is what I had in mind. In fact, I imagine it would reinforce that content, making it more apparent that the code shouldn't be used, in contrast to the code that has a copy widget. 👍

Reply:
@ddbeck I like it, thanks again for the feedback!

Reply:
> including those meant to illustrate something negative

I don't think it's bad to have the clippy icon on those too. You might want to copy it to test and see that you get the nasty error that you expect to get.

Reply:
@peterbe I didn't think it seemed that bad to keep the clippy icon on those around either. Going to try copying it now with all the updated changes from yours and schalks most recent review. Will update shortly.

Reply:
After trying to copy these code examples locally, I think its fine if they stick around. The functionality works and I don't see any negative effects being produced by them sticking around. 

Things seem to be working as expected. Also, Schalk's style changes are incorporated if things like a tad bit different.

https://user-images.githubusercontent.com/48612525/120696448-6dec3300-c461-11eb-8ead-ed06fb556590.mov

Reply:
@peterbe 

> I don't think it's bad to have the clippy icon on those too. You might want to copy it to test and see that you get the nasty error that you expect to get.

Maybe, but it's unlikely to be worth the cost of facilitating readers copying the wrong example in a bad-good example pair, which they'll do. This is not the first docs project I've been on where we added a copy widget to code samples. You'll never guess what we had to change a few weeks later. 🙃

(I'm not worried about the "Message" box in the video, but I am worried about the red examples below.)